DataConnectors/CEF/cef_gather_info.py

```diff
@@ -0,0 +1,166 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ----------------------------------------------------------------------------
+# This script is used to install CEF agent on a linux machine an configure the
+# syslog daemon on the linux machine.
+# Supported OS:
+#   64-bit
+#       CentOS 7 and 8
+#       Amazon Linux 2017.09
+#       Oracle Linux 7
+#       Red Hat Enterprise Linux Server 7 and 8
+#       Debian GNU/Linux 8 and 9
+#       Ubuntu Linux 14.04 LTS, 16.04 LTS and 18.04 LTS
+#       SUSE Linux Enterprise Server 12, 15
+#   32-bit
+#       CentOS 7 and 8
+#       Oracle Linux 7
+#       Red Hat Enterprise Linux Server 7 and 8
+#       Debian GNU/Linux 8 and 9
+#       Ubuntu Linux 14.04 LTS and 16.04 LTS
+# For more information please check the OMS-Agent-for-Linux documentation.
+#
+
+import time
+import subprocess
+
+
+class SystemInfo:
+    def __init__(self, command, command_output="No output received"):
+        self.command = command
+        self.command_output = command_output
+
+    def __repr__(self):
+        delimiter = "\n-----------------------------------\n"
+        return str(
+            delimiter + "command: " + self.command + '\n' + "output: " + self.command_output + delimiter)
+
+
+script_version = '1.0'
+output_file_path = '/tmp/cef_get_info'
+
+basic_command_dict = {
```

Comment:
why not creating a dictionary here with one value and avoid using 2 dictionaries/running function